### DISCUSSION TITLE
Inputs for DISCON-UMaineSemi.in

### DISCUSSION THREAD

**Discussion by ghost** [NatLabRockies/ROSCO#88] [Q&A] [answered]
Hello. I try to understand DISCON-UMaineSemi. In. 
In the file, it seems that Fl_Mode is set 2 without any explanation of option 2.
Is there anyone who has idea of this input?

**Comment by nikhar-abbas** [comment, marked as answer]
Hi @jinch999,
Thanks for asking this question here - you are correct, it looks like we forgot to update the documentation on this! 

When `Fl_Mode = 1`, the nacelle fore-aft velocity is used [m/s], when `Fl_Mode = 2`, the Nacelle IMU fore-aft velocity is used [rad/s]. Please keep in mind that this does change the tuning process as well (so you should be consistent with the tuning .yaml file) , as the feedback gain changes accordingly. 

We'll be sure to incorporate this fix to the docs soon.

Thanks!
Nikhar

**Reply by ghylander** [reply]
I understand then that Fl_mode = 2 is already implemented?
Both for controller tuning and controller usage?

**Reply by nikhar-abbas** [reply]
Yes, as of v2.4.0 this capability is implemented.
